lambda/repository/ingestion_job_repo.py

```
@@ -165,3 +187,81 @@ def list_jobs_by_repository(
         last_evaluated_key_response = response.get("LastEvaluatedKey")
 
         return jobs, last_evaluated_key_response
+
+    def get_batch_job_status(self, job_id: str) -> Optional[str]:
+        """Get the status of a batch job by job ID.
+
+        Args:
+            job_id: AWS Batch job ID
+
+        Returns:
+            Job status (SUBMITTED, PENDING, RUNNABLE, STARTING, RUNNING, SUCCEEDED, FAILED) or None
+        """
+        try:
+            response = self.batch_client.describe_jobs(jobs=[job_id])
+            if response.get("jobs"):
+                return response["jobs"][0].get("status")
+        except Exception:
+            pass
+        return None
```

I assume the intention is if the job doesn't exist it should return `None`, but this is hiding any other exception that could occur. I don't love this design but at a minimum we should be logging this exception.

---

Removed the swallowed exception catch